roles/test_operator/tasks/run-test-operator-job.yml

```diff
@@ -67,15 +67,43 @@
     - not cifmw_test_operator_dry_run | bool
     - not testjob_timed_out
   block:
-    - name: Get information about PVC that stores the logs
+    - name: Get information about PVCs that store the logs
       kubernetes.core.k8s_info:
         kubeconfig: "{{ cifmw_openshift_kubeconfig }}"
         api_key: "{{ cifmw_openshift_token | default(omit)}}"
         context: "{{ cifmw_openshift_context | default(omit)}}"
         kind: PersistentVolumeClaim
         label_selectors:
           - "instanceName={{ test_operator_job_name }}"
-      register: logsPVC
+      register: logsPVCs
+
+    - name: Set up empty volume mounts and volumes
```

Comment:
What do you think about removing this task and using `default([])` in the task below?

Reply:
Done, good idea.